opentechinstitute/commotion-docs
# Updated Developer Resources & Downloads for 1.1

Minimal update to address Commotion-Router developer resources per #40. 
- Updated links on developer resources page.
- Added subpages for commotion-router and commotion-android developers
- Added link to source code and build instructions on commotion-router downloads page

To test:
1. Read for clarity and typos
2. Check links
3. @critzo Make sure the updates didn't break the new publish system.
